xgboost_synthetic/build-train-deploy.ipynb

```diff
@@ -27,22 +27,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Seems you need to add <code>--user</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
added